### PR TITLE
#496

### LINKED ISSUE #496: On request detail, move comment tabs under the info tab
Move the content of the comments tab under the info tab. 
Please create a new section (similar to the FAQ section) to store the comments

Example :

### PR DESCRIPTION
**Pull request by rsercano**
fixes #496

### CODE REVIEW

File: tweb/src/main/resources/templates/pages/requests/detail.html [hunk 1]
```diff
@@ -108,43 +106,59 @@ <h1 class="mb-2 media">
...
+                <div class="col-12">
+                    <h2 class="section-title">Comments</h2>
+                    <h3 class="section-subtitle">
+                        Comments posted on the request
+                    </h3>
+                </div>
+                <div class="col-12">
+                    <div class="card comment">
+                        <div class="card-body">
+                            <div th:each="comment : ${githubComments}">
```
Comment (Simsteve7) [inline tweb/src/main/resources/templates/pages/requests/detail.html]: the th:each should be on the ```<div class="card comment">```. (See difference on you local machine, and production)

Comment (rsercano) [reply tweb/src/main/resources/templates/pages/requests/detail.html]: checking it :)

Comment (rsercano) [reply tweb/src/main/resources/templates/pages/requests/detail.html]: fixed Steve, sorry for it

**Comment by Simsteve7** [comment]
@karelstriegel ready to test

**Pull request merged**